rlm_python/Dockerfile

```diff
@@ -1,33 +1,21 @@
-ARG BASE_IMAGE=opensuse/tumbleweed:latest
-FROM ${BASE_IMAGE} AS repos
-RUN \
-    --mount=type=cache,id=zypp,target=/var/cache/zypp \
-    zypper mr -k repo-oss && \
-    zypper mr -k repo-update
-
-# ======================
-FROM repos
-
+FROM freeradius/freeradius-server:latest
 EXPOSE 1812 1813
+ARG RADIUS_USER=freerad
+ENV TZ=Etc/UTC
```

Comment:
I think we shouldn't embed the TZ at build here, but recommend people bind mount it in from the container host `"/etc/localtime:/etc/localtime:ro"`

Reply:
It's an unavoidable part of the build process - if you install tzdata, you have to select a timezone and there's no other way of fixing it 😡

Reply:
:(